tests/vse_sync_pp/analyzers/test_gnss.py

```diff
@@ -526,6 +536,8 @@ class TestTimeDeviationAnalyzer(TestCase, metaclass=AnalyzerTestBuilder):
                     'stddev': 0,
                     'variance': 0,
                 },
+                'tdev_samples': np.array([0.]).tolist(),
```

Comment:
You don't need to wrap these in numpy arrays to then turn them back into a list.